src/main/java/io/strimzi/kafka/bridge/http/converter/HttpJsonMessageConverter.java

```diff
@@ -25,6 +28,7 @@
         Integer partitionFromBody = null;
         byte[] key = null;
         byte[] value = null;
+        MultiMap headers = MultiMap.caseInsensitiveMultiMap();
```

Comment:
why not filling a `List<KafkaHeader>` that the Vert.x client provides and then adding the list to the record at the end instead of filling this map and then traversing it for adding one header at time?

Reply:
Indeed, I have update the PR accordingly.